msi/docs/build-msi-locally.md

```diff
@@ -0,0 +1,129 @@
+Should you ever need to rebuild a Jenkins MSI on your Windows machine, here is a way to do it.
+
+# Pre-requisites
+
+## Jenkins WAR file
+
+First of all, you should get the Jenkins war file that will be inside that MSI file.
+You can get it from the official Jenkins website or from the Jenkins update center.
+
+Check [Jenkins download page](https://www.jenkins.io/download/) and download the latest weekly version of Jenkins for example.
+Currently, it is [jenkins.war](https://get.jenkins.io/war/2.380/jenkins.war) version 2.380.
+
+Change the URL according to the version you want to download.
+
+## Git
+
+There are quite a few ways to get Git on Windows, but the most straightforward is to see what the [official Git website recommends](https://git-scm.com/download/win).
+
+## Install MSBuild
+
+You can install [MSBuild](https://aka.ms/vs/17/release/vs_BuildTools.exe) from Visual Studio or from the [Build Tools for Visual Studio](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022).
+
+This command line tool is used to build the MSI file.
+
+## Install .NET Framework 3.5
+
+You may already have it installed on your machine, but not activated.
+You can activate it from the Windows Features dialog box.
+
+To access this dialog box, smash <kbd>⊞ Win</kbd> + <kbd>R</kbd>, then enter the command `appwiz.cpl` and push enter.
+Search for
+
+> Turn Windows features on or off.
+
+Tick the `.NET Framework 3.5` entry and install.
+
+Important: now run Windows Update to check for security updates.
+
+If it is not installed yet, you can install [.NET Framework 3.5](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net35-sp1) from the Windows Features.
+
+## Check you have Powershell 5.1
```

Comment:
```suggestion
### Check if you have Windows PowerShell 5.1
```

Reply:
What if they have PowerShell 7.x? Does it change anything?

FTR: https://learn.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.3

Reply:
I unfortunately have no idea.